src/openzaak/components/documenten/tests/test_auth.py

```diff
@@ -376,6 +376,40 @@ def test_eio_create_with_catalogus_autorisatie(self):
                 response.status_code, status.HTTP_201_CREATED, response.data
             )
 
+        with self.subTest("without vertrouwelijkheidaanduiding"):
+            response = self.client.post(
+                url,
+                {
+                    "informatieobjecttype": f"http://testserver{reverse(self.informatieobjecttype)}",
+                    "bronorganisatie": "517439943",
+                    "creatiedatum": "2018-12-24",
+                    "titel": "foo",
+                    "auteur": "bar",
+                    "taal": "nld",
+                },
+            )
+
+            self.assertEqual(
+                response.status_code, status.HTTP_201_CREATED, response.data
+            )
+
+        with self.subTest("without informatieobjecttype"):
+            response = self.client.post(
+                url,
+                {
+                    "vertrouwelijkheidaanduiding": VertrouwelijkheidsAanduiding.openbaar,
+                    "bronorganisatie": "517439943",
+                    "creatiedatum": "2018-12-24",
+                    "titel": "foo",
+                    "auteur": "bar",
+                    "taal": "nld",
+                },
+            )
+
+            self.assertEqual(
+                response.status_code, status.HTTP_400_BAD_REQUEST, response.data
+            )
```

Comment:
Could you also add an assertion on the error code by using `get_validation_errors`?